clash-cores/src/Clash/Cores/Xilinx/Floating/Internal.hs

```diff
@@ -4,17 +4,25 @@ License    :  BSD2 (see the file LICENSE)
 Maintainer :  QBayLogic B.V. <devops@qbaylogic.com>
 -}
 
+{-# LANGUAGE LambdaCase #-}
+
 module Clash.Cores.Xilinx.Floating.Internal
   ( Config(..)
   , ArchOpt(..)
   , DspUsage(..)
   , BMemUsage(..)
+  , Ordering(..)
   , xilinxNaN
   , conditionFloat
   , conditionFloatF
+  , xilinxCompare
+  , toMaybeOrdering
   ) where
 
-import Clash.Prelude
+import Clash.Prelude hiding (Ordering(..))
+import qualified Prelude as P
+
+import Clash.Annotations.BitRepresentation.Deriving
```

Comment:
I have interpreted our style guide to mean that you need an explicit import list here because `clash-prelude` is a standard library. YMMV,